src/test/shared/clients/defaultLambdaClient.test.ts

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DefaultLambdaClient, LambdaClient } from '../../../shared/clients/lambdaClient'
+import { stub, createStubInstance, SinonStubbedInstance } from 'sinon'
+import { Lambda, Request } from 'aws-sdk'
+import * as assert from 'assert'
+import { FunctionUrlConfigList, FunctionUrlConfig, ListFunctionUrlConfigsResponse } from 'aws-sdk/clients/lambda'
+
+describe('Test the default lambda client', async () => {
+    let client: LambdaClient
+    const regionCode: string = 'my-region-code'
+    const lambdaFuncName = 'lambda-func-name'
+    let lambdaApiMock: SinonStubbedInstance<Lambda>
+
+    beforeEach(async () => {
+        lambdaApiMock = stub(new Lambda())
+        client = new DefaultLambdaClient(regionCode, lambdaApiMock as any)
```

Comment:
Instead of the "any" cast, is it possible to cast to Lambda? Or does that cause an error?

Reply:
It gave an error, but I found another way that I've updated it to `<Lambda><unknown> lambdaApiMock` which converts it to an 'unknown' then the type we want. 

Source: https://github.com/microsoft/TypeScript/issues/28067